languages-and-frameworks/axum.html.markerb

```diff
@@ -0,0 +1,115 @@
+---
+title: "Run a Rust Axum App"
+layout: framework_docs
+sitemap: false
+order: 1
+---
+
+<% app_name = "hello-next" %>
+<%= partial "/docs/js/frameworks/partials/intro", locals: { runtime: "Rust Axum", link: "https://docs.rs/axum/latest/axum/" } %>
```

Comment:
This partial is specific to Javascript, but there's one with the same content in the general Frameworks directory here that you could use instead: https://github.com/superfly/docs/blob/main/languages-and-frameworks/partials/_intro.html.erb

`partial "partials/intro"`